packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -10,4 +10,25 @@ const designSystemDefaults: IDesignSystem = {
     brandColor: "#0078D4"
 };
 
+/**
+ * Returns RGBA from hex value with optional alpha

```

Comment:
we should pull this into a utilities folder if we're going to use this one. We may also want to just interpolate the values below

Reply:
Also - questions as to whether or not this should live here or another package - colors, jss-utiltites...likely a team convo. But I'd pull from the design system folder.

Reply:
We probably need to figure out the full color story or there is going to be a lot of redoing work. I could have simply added the rgba values, but this seems like something we would want anyway. I can move, but we should figure this out as a team.

Reply:
Agree on syncing as a team here. Not saying it needs to land, but I think we can start categorizing in utils for now at the very least.

Reply:
Ok @nicholasrice, I see a function think I can use from chroma-js to accomplish this. Do I need to add chroma to the styles-msft repo to use?